YDS-Storybook/SwiftUI/Atom/BadgePageView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BadgePageView.swift
+//  YDS-Storybook
+//
+//  Created by 성현주 on 10/31/23.
+//
+
+import SwiftUI
+import YDS_SwiftUI
+
+
+
+struct BadgePageView: View {
+    let title: String = "BadgeView"
+    
+    @State var text: String? = "광고"
+    @State var badgeColorSelectedIndex = 0
+    @State var icons: SwiftUIIcon? = YDSSwiftUIIcon.icons[0]
+    
+    var selectedColor: Color {
+        return YDSItemColor.allCases[badgeColorSelectedIndex].backgroundColor
+    }
+
+    
+    public var body: some View {
```

Comment:
이 부분도요!
+ 다른 부분도 탭이 들어가 있어서 경고가 뜨는 부분이 있어요. 한 번 확인해주세요

```suggestion
    }

    public var body: some View {
```